storybook/src/Storybook/Story.elm

```diff
@@ -0,0 +1,33 @@
+module Storybook.Story exposing (Story, sandbox, stateless)
+
+import Browser
+import Html exposing (Html)
+
+
+type alias Story model msg =
+    Platform.Program () model msg
+
+
+stateless : { view : Html msg } -> Story () msg
+stateless options =
+    Browser.element
+        { init = \() -> ( (), Cmd.none )
+        , update = \msg model -> ( model, Cmd.none )
```

Comment:
I think it's nice to keep `logAction` here so that actions appear on the storybook UI.
https://github.com/yoching/unison-ui-core-storybook/blob/main/src/Storybook/Story.elm#L18

Reply:
I added it again. I needed to add the keyword `port` to the module declaration to make it work